code/datums/uplink/highly visible and dangerous weapons.dm

```diff
@@ -183,3 +183,78 @@
 	name = "Energy Pistol"
 	item_cost = 2
 	path = /obj/item/gun/energy/pistol
+
+/datum/uplink_item/item/visible_weapons/gyrojet_pistol
+	name = "Gyrojet Pistol"
+	item_cost = 20
+	path = /obj/item/gun/projectile/gyropistol
+
+/datum/uplink_item/item/visible_weapons/bullpup_assault_carbine
+	name = "Bullpup Assault Carbine"
+	item_cost = 12
+	path = /obj/item/gun/projectile/automatic/rifle/z8
+
+/datum/uplink_item/item/visible_weapons/assault_shotgun
+	name = "Assault Shotgun"
+	item_cost = 15
+	path = /obj/item/gun/projectile/automatic/rifle/shotgun
+
+/datum/uplink_item/item/visible_weapons/gatling_laser
+	name = "Gatling Laser"
+	item_cost = 40
+	path = /obj/item/gun/energy/vaurca/gatlinglaser
+
+/datum/uplink_item/item/visible_weapons/laser_shotgun
+	name = "Laser Shotgun"
+	item_cost = 10
+	path = /obj/item/gun/energy/laser/shotgun
+
+/datum/uplink_item/item/visible_weapons/xray_laser
+	name = "X-Ray Laser Rifle"
+	item_cost = 12
+	path = /obj/item/gun/energy/rifle/laser/xray
+
+/datum/uplink_item/item/visible_weapons/combat_shotgun
+	name = "Combat Shotgun"
+	item_cost = 10
+	path = /obj/item/gun/projectile/shotgun/pump/combat
+
+/datum/uplink_item/item/visible_weapons/automatic_45_pistol
+	name = "Automatic .45 Pistol"
+	item_cost = 5
```

Comment:
The .45 auto submachine gun has similar stats and that costs 8 TC. I think you should raise the price of this or lower the price of the .45 auto SMG.

Reply:
Capacity for the machine pistol is lower and it can’t use drum magazines